subgraph-js/src/directives.ts

```diff
@@ -88,7 +107,7 @@ export function isFederationDirective(directive: GraphQLDirective): boolean {
   return federationDirectives.some(({ name }) => name === directive.name);
 }
 
-export const otherKnownDirectives = [TagDirective];
+export const otherKnownDirectives = [TagDirective, ContactDirective];
```

Comment:
It looks like this can have the impact of making composition's `printCoreDirectives` print a broken link to https://specs.apollo.dev/contact/v0.1

Reply:
Good catch. Made some small adjustments to how the `ContactDirective` is included and how subgraph directive usages are printed.